contracts/ICometRewards.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.15;
+
+/**
+ * @dev Interface for interacting with the CometRewards contract
+ */
+interface ICometRewards {
```

Comment:
What naming convention should we use here? For most interfaces, we go with the `IContract` convention (e.g. `IComp`), but for Comet, we named it `CometInterface`.

Reply:
We only need like one of these functions for Bulker right? Would it be terrible to just inline the interface we need at the top of Bulker and/or call it something like IClaimable?

Reply:
Yeah, I think inlining is the best probably. Will make that change